index.html

```diff
@@ -158,64 +158,65 @@ <h1 id="title">Dataset Exchange Working Group Charter</h1>
+        <p>A success of a previous WG charter was the publication of DCAT-3, where the evolved through multiple drafts to improve clarity, extend functionality, and align with best practices by introducing dataset series as first-class entities, refining versioning and inverse property handling, enhancing multilingual and accessibility support, improving alignment with external standards, and strengthening guidance on security, privacy, and metadata usage. </p>
 
-        <p>DCAT has known gaps in coverage, for example around time series and versions. DCAT has been successful and is in wide use, but these gaps must be addressed if usage is to continue to grow across different communities and the variety of metadata schemas is to reduce.</p>
+        <p>However, despite previous improvements around time series and versions, DCAT has known gaps in coverage, for example in describing the variables encoded within datasets in an explicitily and semantically interoperable manner. Variables are a fundamental concept in both dataset design and dataset search, partly covered in existing recommendations like the <a href="https://www.w3.org/TR/tabular-metadata/">Metadata Vocabulary for Tabular Data</a> (used to annotate tabular data) and the <a href="https://www.w3.org/TR/vocab-data-cube/">RDF Data Cube Vocabulary</a> (used to describe statistical data and time series). While these recommendations have been successful and are in wide use, this gap must be addressed if usage is to continue to grow across different communities and the variety of metadata schemas is to reduce.</p>
```

Comment:
Minor changes (suggestions):
```suggestion
        <p>However, despite all those improvements, DCAT has known gaps in coverage, for example in describing the variables encoded within datasets in an explicitily and semantically interoperable manner. Variables are a fundamental concept in both dataset design and dataset search, partly covered in existing recommendations like the <a href="https://www.w3.org/TR/tabular-metadata/">Metadata Vocabulary for Tabular Data</a> (used to annotate tabular data) and the <a href="https://www.w3.org/TR/vocab-data-cube/">RDF Data Cube Vocabulary</a>, QB, (used to describe statistical data and time series). While these recommendations have also been successful and are in wide use, this gap must be addressed if usage is to continue to grow across different communities and the variety of metadata schemas is to be reduced.</p>
```